receiver/sqlqueryreceiver/integration_test.go

```diff
@@ -112,32 +113,42 @@ var (
 		},
 	}
 	Oracle = DbEngineUnderTest{
-		Port:         oraclePort,
-		SQLParameter: ":1",
+		Port: oraclePort,
+		SQLParameter: func(position int) string {
+			return fmt.Sprintf(":%d", position)
+		},
 		CheckCompatibility: func(t *testing.T) {
-			if runtime.GOARCH == "arm64" {
-				t.Skip("Incompatible with arm64")
-			}
 			t.Skip("Skipping the test until https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27577 is fixed")
 		},
 		ConnectionString: func(host string, externalPort nat.Port) string {
-			return fmt.Sprintf("oracle://otel:p@ssw%%25rd@%s:%s/XE", host, externalPort.Port())
+			return fmt.Sprintf("oracle://otel:otel@%s:%s/FREEPDB1", host, externalPort.Port())
 		},
-		Driver:                   "oracle",
-		CurrentTimestampFunction: "SYSTIMESTAMP",
-		ConvertColumnName:        strings.ToUpper,
+		Driver:            "oracle",
+		ConvertColumnName: strings.ToUpper,
 		ContainerRequest: testcontainers.ContainerRequest{
-			FromDockerfile: testcontainers.FromDockerfile{
-				Context:    filepath.Join("testdata", "integration", "oracle"),
-				Dockerfile: "Dockerfile.oracledb",
+			Image: "gvenzl/oracle-free:slim-faststart",
```

Comment:
Just commenting for future reference: I think using the latest version works for me as it will show us if we ever break compatibility with the latest upstream version. If this test ends up failing we can fix the incompatibility and pin the version in the meanwhile.